readme.md

```diff
@@ -184,7 +169,209 @@ Working directory from which to resolve a `contributors` module, if any
+```sh
+npm run format
+```
+
+### Example: `appendIfMissing`
+
+The default behavior of this plugin is to not generate lists of Git
+contributors if there is no `## Contributors` (case- and level-insensitive).
+You can change that by configuring the plugin with
+[`appendIfMissing: true`][append-if-missing].
+
+The reason for not generating contributors by default is that as we saw in the
+previous example (CLI in npm scripts) remark and this plugin often run on
+several files.
+You can choose where to add the list by explicitly adding `## Contributors`
+in the main file (`readme.md`) and other docs won’t be touched.
+Or, when you have many contributors, add a specific `contributors.md` file,
+with a primary `# Contributors` heading, and the list will be generated there.
+
+To turn `appendIfMissing` mode on, pass it like so on the API:
+
+```js
+  // …
+  .use(remarkGitContributors, {appendIfMissing: true})
+  // …
+```
+
+Or on the CLI (in `package.json`):
+
+```js
+  // …
+  "remarkConfig": {
+    "plugins": [
+      // …
+      [
+        "remark-git-contributors",
+        {"appendIfMissing": true}
+      ]
+    ]
+  },
+  // …
+```
+
+### Example: metadata
+
+The data gathered from Git is somewhat sparse.
+To add more metadata, either add it to `package.json` (used in this project’s
+[`package.json`][package-json]) or configure `options.contributors`.
+On the API, that’s done like so:
+
+```js
+  // …
+  .use(remarkGitContributors, {contributors: /* value */})
+  // …
+```
+
+Or on the CLI (in `package.json`):
+
+```js
+  // …
+  "remarkConfig": {
+    "plugins": [
+      // …
+      [
+        "remark-git-contributors",
+        {"contributors": /* value */}
+      ]
+    ]
+  },
+  // …
+```
+
+The value for `contributors` is either:
+
+*   an array in the form of `[{ email, name, … }, … ]`;
+*   a module id, or path to a file, that exports `contributors` as the default
+    export or as a `contributors` named export
+
+> 👉 **Note**: contributors that are not in Git history are excluded.
+> This way the `contributors` metadata can be reused in multiple projects.
+
+Each contributor should at least have an `email` property to match against Git
+email addresses.
+If you’re experiencing people showing up multiple times from Git history, for
+example because they switched email addresses while contributing to the project,
+or if their name or email are wrong, you can “merge” and fix contributors in Git
+by using a [`.mailmap` file][mailmap].
+
+The supported properties on contributors are:
+
+*   `name` — person’s name (example: `Sara`)
+*   `email` — person’s email (example: `sara@example.com`)
+*   `github` — GitHub username (example: `sara123`)
+*   `twitter` — Twitter username (example: `the_sara`)
+*   `mastodon` — Mastodon (`@user@domain`)
+
+An example of a reusable module is [`level-community`][level-community] by the
+`level-js` community.
+
+```js
+  // …
+  .use(remarkGitContributors, {contributors: 'level-community'})
+  // …
+```
```

Comment:
has this been tested since the ESM move?

Reply:
I (as maintainer of Level) am no longer using this. This being `remark-git-contributors`, `remark-contributors` and `level-community`. So probably best to remove or replace the example (and perhaps the feature itself) with something up to date

Reply:
yep, works.